packages/examples/src/validated-generation.tsx

```diff
@@ -10,21 +13,51 @@ const FamilyTree: z.Schema = z.array(
   })
 );
 
+const RootFamilyTree: z.Schema = z.object({
+  tree: FamilyTree,
+});
+
 function App() {
   const query = 'Create a nested family tree with names and ages. It should include a total of 5 people.';
   return (
     <>
       JSON generation example:{'\n'}
-      <JsonChatCompletion schema={FamilyTree}>
-        <UserMessage>{query}</UserMessage>
-      </JsonChatCompletion>
-      {'\n\n'}
+      <ChatProvider model="gpt-4">
```

Comment:
To make this work, I had to switch to a model that supports functions.

Reply:
doesn't gpt 3.5 support function calls?

Reply:
Either way consider `<OpenAI chatModel="...">` instead

Reply:
Ah, looks like it does actually.